boom! testing checked_cast-0.0.3 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! 24527fd6d95b061a6ffc3bb80c41773bd82082bdf33cee4a3fab0ac4018db522
boom! running `docker start -a 24527fd6d95b061a6ffc3bb80c41773bd82082bdf33cee4a3fab0ac4018db522`
kablam!    Compiling checked_cast v0.0.3 (file:///source)
kablam! warning: unknown lint: `unstable`
kablam!  --> src/lib.rs:3:10
kablam!   |
kablam! 3 | #![allow(unstable)]
kablam!   |          ^^^^^^^^
kablam!   |
kablam!   = note: #[warn(unknown_lints)] on by default
kablam! 
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.26 secs
boom! running `docker rm -f 24527fd6d95b061a6ffc3bb80c41773bd82082bdf33cee4a3fab0ac4018db522`
blam! 24527fd6d95b061a6ffc3bb80c41773bd82082bdf33cee4a3fab0ac4018db522
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 8f66b02e392ac153b4606c07614a5334390a8a081b17167309fb15187f1f1e02
boom! running `docker start -a 8f66b02e392ac153b4606c07614a5334390a8a081b17167309fb15187f1f1e02`
kablam!    Compiling checked_cast v0.0.3 (file:///source)
kablam! warning: unknown lint: `unstable`
kablam!  --> src/lib.rs:3:10
kablam!   |
kablam! 3 | #![allow(unstable)]
kablam!   |          ^^^^^^^^
kablam!   |
kablam!   = note: #[warn(unknown_lints)] on by default
kablam! 
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.83 secs
boom! running `docker rm -f 8f66b02e392ac153b4606c07614a5334390a8a081b17167309fb15187f1f1e02`
blam! 8f66b02e392ac153b4606c07614a5334390a8a081b17167309fb15187f1f1e02
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! f75416b224e34748e65c0fc91a686c5de55fed3e917b0d4fc55abe215447fc51
boom! running `docker start -a f75416b224e34748e65c0fc91a686c5de55fed3e917b0d4fc55abe215447fc51`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/checked_cast-7074f26665ab07aa
blam! 
kablam!    Doc-tests checked_cast
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
blam! 
blam! running 1 test
blam! test /source/src/lib.rs - core (line 7) ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- /source/src/lib.rs - core (line 7) stdout ----
blam! 	error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:18:40
blam!    |
blam! 18 |     let acast: Option<libc::c_ulong> = checked_cast!(a, libc::c_ulong);
blam!    |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:18:40
blam!    |
blam! 18 |     let acast: Option<libc::c_ulong> = checked_cast!(a, libc::c_ulong);
blam!    |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:27:38
blam!    |
blam! 27 |     let bcast: Option<libc::c_int> = checked_cast!(b, libc::c_int);
blam!    |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:27:38
blam!    |
blam! 27 |     let bcast: Option<libc::c_int> = checked_cast!(b, libc::c_int);
blam!    |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:35:39
blam!    |
blam! 35 |     let ccast: Option<libc::c_uint> = checked_cast!(c, libc::c_uint);
blam!    |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:35:39
blam!    |
blam! 35 |     let ccast: Option<libc::c_uint> = checked_cast!(c, libc::c_uint);
blam!    |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0412]: cannot find type `int` in this scope
blam!   --> <anon>:42:12
blam!    |
blam! 42 |     let d: int = 2314;
blam!    |            ^^^ not found in this scope
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:43:29
blam!    |
blam! 43 |     let dcast: Option<u8> = checked_cast!(d, u8);
blam!    |                             ^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:43:29
blam!    |
blam! 43 |     let dcast: Option<u8> = checked_cast!(d, u8);
blam!    |                             ^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:53:22
blam!    |
blam! 53 |     let ecast: i16 = checked_cast!(e, i16).unwrap();
blam!    |                      ^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error[E0433]: failed to resolve. Could not find `Int` in `num`
blam!   --> <anon>:53:22
blam!    |
blam! 53 |     let ecast: i16 = checked_cast!(e, i16).unwrap();
blam!    |                      ^^^^^^^^^^^^^^^^^^^^^ Could not find `Int` in `num`
blam!    |
blam!    = note: this error originates in a macro outside of the current crate
blam! 
blam! error: The attribute `should_fail` is currently unknown to the compiler and may have meaning added to it in the future (see issue #29642)
blam!   --> <anon>:49:1
blam!    |
blam! 49 | #[should_fail]
blam!    | ^^^^^^^^^^^^^^
blam!    |
blam!    = help: add #![feature(custom_attribute)] to the crate attributes to enable
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/src/librustc/session/mod.rs:203
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
kablam! error: test failed
blam! 
blam! failures:
blam!     /source/src/lib.rs - core (line 7)
blam! 
blam! test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured
blam! 
boom! running `docker rm -f f75416b224e34748e65c0fc91a686c5de55fed3e917b0d4fc55abe215447fc51`
blam! f75416b224e34748e65c0fc91a686c5de55fed3e917b0d4fc55abe215447fc51
